meta_definitions/device/definitions.py

```diff
@@ -0,0 +1,3072 @@
+# -*- coding: utf-8 -*-
+from metrics import *
+
+ALL = {
```

Comment:
it's awesome to have these there as well! I only have one comment regarding the encapsulation of all the device JSON schemas inside the ALL variable. If for any reason we need to access only one of the schema definitions, it would be nice not to have to load this big dictionary into memory to do so. What do you think of removing the `ALL` encapsulation?

Reply:
You mean assign each `value` of this `dict` to it's own variable and reference _it_ in this dictionary?